tools/security-tools.MD

```diff
@@ -0,0 +1,59 @@
+# Security Guild recommended tools
+
+This document lists the tools that are available and researched for performing linting, Static Code Analysis, vulnerability scanning on different platforms and languages. The goal is to introduce one of more of these tools into StoneSoup.
+
+From our discussions in the Secure Engineering Guild, our current position is that we would encourage teams to use the tools highlighted in this list for consistency. But if a team finds a different tool that's a better fit-for-purpose, that is also fine. The ProdSec org also supports some tools, and some tools will be integrated into StoneSoup over time (taking the responsibility of implementation off of our teams). So where we have a recommendation, there it also makes sense to follow it if it fits your team/product's needs.  
+ProdSec recommends us to not use tools that could potentially "phone home" or any that could otherwise expose information about embargoed vulnerabilities unless they have previously approved our particular use of the tool.
+
+### Linting Tools
+
+#### Golang
+**golangci-lint** - https://github.com/golangci/golangci-lint
+
+```
+Usage:
+$ cd my/golang/project
+$ golangci-lint run
+```
+
+#### Docker
+**hadolint** - https://github.com/hadolint/hadolint  
+_Note: In order to lint Bash code inside a Dockerfile, it uses Shellcheck internally._  
+  
+```
+Usage:
+$ hadolint Dockerfile  
+(or)  
+$ docker run --rm -i ghcr.io/hadolint/hadolint < Dockerfile
+```
+
+### Vulnerability Scanners
+
+**clair** - https://github.com/quay/clair  
+- quay.io uses Clair internally and the project is officially under them. 
+- Check these [docs](https://quay.github.io/clair/howto/deployment.html) to understand the deployment models Clair currently uses which also talks about on setting it up in an OpenShift or a Kubernetes environment. 
```

Comment:
I recommend dropping "which also talks about on setting it up in an OpenShift or a Kubernetes environment". Otherwise make this a couple of sentences for readability.